src/containers/GlobalInterceptors/index.js

```diff
@@ -0,0 +1,40 @@
+import { useContext, useEffect } from 'react';
+import GlobalErrContext from '../../context/GlobalErrContext';
+import axios from 'axios';
+import humps from 'humps';
+
+const GlobalInterceptors = () => {
```

Comment:
Why is this a component. I don;t understand what it does. I mean, why a component would intercept axios responses???

Reply:
We decided to go on with this instead of using helper files and importing them. What kind of harm it can do? Acts like a container.

Reply:
Please fix. A component shouldn't do what api layer is responsible for